pkg/config/config.go

```diff
@@ -62,6 +62,7 @@ type Bundle struct {
 	DB         string   `yaml:"db_path,omitempty"`
 	LocalFile  bool     `yaml:"local_file,omitempty"`
 	Targets    []string `yaml:"targets,omitempty"`
+	Upgrade    bool     `yaml:"upgrade,omitempty"`
```

Comment:
I don't think we really need an "upgrade" option here, but rather make the install idempotent and upgrade the bundle. Also, having an upgrade setting for a first-deployment would be quite odd

Reply:
>  make the install idempotent and upgrade
how do I go about this one 

`luet search` for the package and if its already present switch to `luet upgrade` ?